doc/mkdocs.yml

```diff
@@ -0,0 +1,12 @@
+
+#- ['distribution/index.md', 'Registry v2', 'User Guide']
+
+- ['distribution/auth/token.md', 'Registry v2', 'Token']
```

Comment:
The original registry docs also need to be added to this menu structure - so we need to differentiate them.